### DISCUSSION TITLE
Experiments: First Run (Depth 0)

### DISCUSSION THREAD

**Discussion by algomaster99** [ASSERT-KTH/collector-sahab#214] [General]
The following commits did not have a render HTML output for them. The experiment was run using repairnator machine which is running Java `11` and Maven `3.6.3`. 

|                    Patch                                                            |  Reason  |
|--------------------------------------------------------------------------------|---|
| patch2-Math-73-JGenProg2017_25c92fbe9ffd72eac6d4829ceb4abcc4a4b5c86e           | https://github.com/ASSERT-KTH/collector-sahab/issues/215  |
| patch3-Math-40-Cardumen_e979e14806909f944bc0c70b91ee530657d2ab26               | https://github.com/ASSERT-KTH/collector-sahab/issues/215#issuecomment-1526517638   |
| patch1-Math-70-ssFix_0a682bad1e0b361ffa20b9382203114c4054b88c                  | https://github.com/ASSERT-KTH/collector-sahab/issues/215#issuecomment-1526521898   |
| patch1-Math-50-Jaid-plausible_0e9b735aad6cbd5892ee16da01e8c6d39111eac5         | #215   |
| patch3-Time-17-Cardumen_f8431b4cbe4570108789c8ad6a23f02cab486b05               | https://github.com/ASSERT-KTH/collector-sahab/issues/216  |
| patch1-Math-70-Arja_866f0393248eac6408e4a3ea7eb5a5c1bbfb9853                   | #215    |
| patch1-Math-56-Arja-plausible_00cee0590e591a074a879adf2d886f2e0911fc7a         | https://github.com/ASSERT-KTH/collector-sahab/issues/217  |
| patch1-Math-84-jKali_615afa846ea8fc7752457076f4cb579616d45724                  |  Invisible Diff  |
| patch1-Math-41-ssFix_1d066b30a6d441b256cfa32034be73fbecb18b89                  | #215   |
| patch3-Math-70-Cardumen_30709758f0b33338fa697b400112fac836e4cef1               | https://github.com/ASSERT-KTH/collector-sahab/issues/219  |
| patch3-Math-30-Cardumen_b601a7fa9e7ede49b8acddfe73a73ca41ca4451c               | #215  |
| patch5-Math-84-Cardumen_4a88e54644f007e5635a00e349b142bde490b939               |  Invisible Diff  |
| patch1-Math-50-JGenProg2015_44b1a99ac1fbafdbeac01e337e0c191472fe3d37           | #215    |
| patch3-Math-2-JGenProg2017_0447cfda4187d85e9909841a51ead067e5ce6257            | #218   |
| patch7-Math-63-CapGen-plausible_612c9cc3dc6982c605721c62c899fbaa96498c69       | #218   |
| patch4-Math-97-Cardumen_b80210e1b93b5b074b85069f52ec4ac51b75668d               | https://github.com/ASSERT-KTH/collector-sahab/issues/215  |
| patch4-Math-78-Cardumen_d034728aa5fcb3ee8af50c2ad0118c179fe025e3               | #218   |
| patch2-Math-70-SketchFix-plausible_00a731b690e0db1a5ac1f7158201ca300cd0ae91    | #217    |
| patch4-Math-31-Arja-plausible_9a573745a43b4ea1d5ce939d67179ef13da88a4d         | #215    |
| patch2-Math-2-JGenProg2017_5e800f0767bb3403b129488db48c7d8f5973f7a1            | #218    |
| patch3-Math-41-Cardumen_8b1372284ef2b5d53c029f2f84d7a7815c11b1e6               |  #215   |
| patch5-Math-31-Arja-plausible_f4c53b9b7cff5492eb5b58799694a54cf828f6f9         |  #215   |
| patch2-Math-104-Cardumen_98dc9eef38a8a3353d110786ffed93ec83d1cc02              | https://github.com/ASSERT-KTH/collector-sahab/issues/220   |
| patch1-Math-63-CapGen-plausible_e12c96d6843e46f5448c964f7a2e32162db82bd7       | #218   |
| patch1-Math-41-SimFix_574c9be19e191fe2dc548fb2b7dff88975402208                 | #215   |
| patch1-Math-35-SimFix_6043bd63b93ee4565b6ef3efe0179fbb2351b9c3                 |   |
| patch2-Math-84-jMutRepair_7aae5f73400a8785cd184849bdd6a9b26a094ff2             |   |
| patch4-Math-84-Cardumen_179274304f3f45ec8e0fe048e4c88c54c8df8f49               |   |
| patch4-Math-46-Cardumen_f4fc61b10d3d4984847815f2f8e08786fa77112a               |   |
| patch3-Math-2-Arja-plausible_b57e8431afde1b7267d4fed84b5271e293b8360c          |   |
| patch2-Math-18-Cardumen_82260afb10c85a927e4a2344352e851d7b5742b6               |   |
| patch2-Time-11-JGenProg2017_396f09be09422635ef116c107da72f47bafdf2b1           |   |
| patch1-Math-63-Elixir-plausible_0cecf45f8e8bf3982244d810acb86dd7dc734931       |   |
| patch1-Math-50-Cardumen_33837fa0d924e774200367acd9af74c391096080               |   |
| patch2-Math-84-Cardumen_822d276d2b732d81b484c7626e86a769da669ec2               |   |
| patch5-Time-4-Arja-plausible_38b1c6e5763b55870959e6d75a6e5b46cacd382e          |   |
| patch1-Time-7-SimFix_b94c887b9579ca7816a897d91dd68f6650383eae                  |   |
| patch1-Math-73-JGenProg2015_9d4a978e476dbe6072fe9fbaef9b1288cc8fa729           |   |
| patch1-Math-84-jMutRepair_81b32ad74037364e7eccdd64728fcb08b03958b9             |   |
| patch3-Math-105-Cardumen_885bbd7d0f6db843627986bf97d3f0db4cfddb02              |   |
| patch1-Math-7-JGenProg2017_329d50c82fd3e024bd8d1f915a3977df321b1336            |   |
| patch1-Math-70-JGenProg2015_468021d2454c7a8852062e644caccc4cbfc5bd00           |   |
| patch1-Math-60-Cardumen_a16399f50f84d3d82d6bb0306996961b2a82bb06               |   |
| patch1-Time-18-Cardumen_972945e85ef38802f44c17cae42a677896eef6e0               |   |
| patch4-Math-18-Cardumen_09e40b014905153aa3ec35974152e4613b6ec885               |   |
| patch1-Math-58-Arja_52b14bbdd2cdbbf9ed1c446f6f87f6ca2e85c5d9                   |   |
| patch4-Math-63-CapGen-plausible_2211883f19f39293bc2c5a55478bb5b6f656ed30       |   |
| patch3-Math-40-JGenProg2017_b5797398eea69ecd61477a1fac649d26002d77d9           |   |
| patch5-Math-30-Cardumen_1eaf9b2a64af6cae31fe2244f21e34b5e0bafb1f               |   |
| patch1-Math-50-HDRepair_3ae853a6464baa3dd498a43e2af9638f8ba09d86               |   |
| patch2-Math-49-JGenProg2017_4227bd7b35bb6ac325767390e9cd907d3a2e8a38           |   |
| patch2-Math-31-Arja-plausible_37757413c6d69946ea68c27c12ac0f7fd38b535c         |   |
| patch1-Math-50-Elixir_d393c448ea8b0e15c5e254635ca344279b01323e                 |   |
| patch2-Math-105-Cardumen_af19a3800c20db42db18bc3ac51ce90236ca9517              |   |
| patch3-Math-84-Cardumen_37039ae28a84fca1c62608b916a5d56c379c6e5c               |   |
| patch2-Math-70-JGenProg2017_1c83da34bb57d7394be1dc06ba2bfe40fac0a1cc           |   |
| patch1-Math-30-Cardumen_1a03ec3d687ede679d10001f33dbd802d8180775               |   |
| patch2-Math-56-Arja-plausible_461a6874efc6d8a5a7c1bedf1bd7c072777529c2         |   |
| patch1-Math-50-ssFix_92a916b1bc372d6f5065d5b22fe460d8c81dbca8                  |   |
| patch1-Time-17-Cardumen_3b47797a8c8938f5aff44cc69409d88a068626e5               |   |
| patch1-Math-63-SequenceR_1fb0e23409228f8bb4802b06445822b8fbdcee76              |   |
| patch8-Math-2-Arja-plausible_a6dfd9ed95a38a625822152d22a97eedddcaa737          |   |
| patch2-Math-84-JGenProg2017_ee7648f61e58794629090bb25acc66b9fe7c82b7           |   |
| patch5-Math-60-Cardumen_fdefab96ec89efaf5e0067970abdcfe8c5324edf               |   |
| patch1-Math-50-JGenProg2017_770a7843e8a2c169cb64fbaca43b8185f29e10d0           |   |
| patch2-Math-60-Cardumen_4958b2b2bf73756def4b272f441b7b1b5812afb0               |   |
| patch1-Math-93-ACS_6389a74884f775da16f85e7b1716a6a52f5ace45                    |   |
| patch1-Math-41-Cardumen_a444e30d929c30cfeb76651624b194aa8ded9be1               |   |
| patch3-Math-60-Cardumen_b941601c972628aa103d6fa353bd25bc579d6993               |   |
| patch1-Time-15-Elixir_c24932ee3f562f66f282ad80d5d9d3a0504917c6                 | https://github.com/ASSERT-KTH/collector-sahab/issues/218  |
| patch1-Math-84-Cardumen_1482e9e4fba71651f86299f5abd0688af176e3db               |   |
| patch1-Time-9-Cardumen_30703bf61a3fba78886876571f92190d1eeff121                |   |
| patch1-Math-35-ACS_d5f817b6b3e6f63e542787f741b4ec8cb62a7662                    |   |
| patch1-Math-63-CapGen_9b7252ca3cf58200ebf805e092c80c0048d5f2bb                 |   |
| patch2-Math-50-Cardumen_62a9df1feb57457d237e63fa3f1d19598cb68a08               |   |
| patch1-Math-18-Cardumen_187de653ef51444790c5edb70180f178570e1902               |   |
| patch1-Time-15-Arja_33524b4563a49a4061d2a400746694533ad83294                   |   |
| patch1-Math-73-JGenProg2017_ddd1d1aec36692137e380b86dcd2df2d68a49b97           |   |
| patch4-Math-63-Cardumen_4fdad82a90df66fefaae358489bd88371a393e97               |   |
| patch1-Math-58-Elixir_574281efda53164c9f7eb8a2a7e58ee4fd390f5b                 |   |
| patch1-Math-57-SequenceR_391c779ea7066607a3c56a1e1ad5bfce761d1e2b              |   |
| patch3-Math-31-Arja-plausible_047ef2fcc23f713dc2e682d8b4f2ca5f994bb7ca         |   |
| patch1-Math-73-SimFix-plausible_8d2f48434c3b53fd7ebb4264c150ab3c06b8a68f       |   |
| patch2-Math-2-jMutRepair_024ec1fee14b5a3683424623c5c4220c1691d7cb              |   |
| patch2-Math-97-Cardumen_09399d94e3aa909eefa4d00d77a68c065a9364a5               |   |
| patch4-Math-2-JGenProg2017_7a4482de9b57a192e23433d8842a5d40134b8bdf            |   |
| patch1-Math-70-HDRepair_0254acc283929f9c50f437c096411eb93a9e89fa               |   |
| patch1-Math-70-Elixir_cedc33a91b8c74fe56dfa3c36648912a8bf343a7                 |   |
| patch1-Math-40-Cardumen_1e39f6dc3ebae81e3d423f8e149625ffb1e0987b               |   |
| patch1-Math-50-jMutRepair_30155ff44e22b7fb62b724e019aa1bb3ce82525c             |   |
| patch2-Math-50-JGenProg2017_47bb0411a0d1bd5740925251e2d6ac6761ee3a5c           |   |
| patch5-Math-46-Cardumen_ac3cc71b7df32e8e5be3843b70e0d738c2f96d47               |   |
| patch5-Math-2-Cardumen_c482262e8e5ee21e9bda3420e8114d4239c1c7fe                |   |
| patch1-Time-15-ACS_6ac4672bf5c560e734e27a6ec9f363ace5961ee1                    |   |
| patch1-Math-46-Cardumen_64ac2ebcdccd46a2a00a95206e028a140095d5d6               |   |
| patch1-Math-20-Elixir-plausible_0a8a5d1d5fc3a6d0681e7344ddd773acdea27dda       |   |
| patch1-Math-50-Jaid_587961ee65b1f949f20da557cdd5b09573b7e483                   |   |
| patch1-Math-81-SimFix-plausible_f21ffef185bb3483f58294cb4aaaafc7940180eb       |   |
| patch2-Math-63-Cardumen_257eacb2789f51d4d915b82553ccb08ede950bed               |   |
| patch3-Math-63-CapGen-plausible_40bc2d6aedfa7cbbb970ac665e3c03826bc68951       |   |
| patch4-Math-50-JGenProg2017_a44c24f57cd24852f44569a80bbb998141e5d0bf           |   |
| patch3-Math-73-Cardumen_fb5372e59b6cf69b47f9c9b7154fb341ef2ad64b               |   |
| patch5-Math-63-CapGen-plausible_8a21d074d895d0c1f9f3f7a8d2d01483b35f3e72       |   |
| patch1-Math-73-Cardumen_a64e73a250ebe61c61f79f36a027e1f8d312ea0d               |   |
| patch1-Math-2-JGenProg2017_da1811cc541f7cc09e980371f3af4488f16ba786            |   |
| patch4-Math-78-JGenProg2017_e29e4f190acd1ff41fc54eac8bb98e502b0d954e           |   |
| patch2-Math-30-Cardumen_e1f7a7580ca5f96e2607b45f0991d06467db1f81               |   |
| patch7-Math-31-Arja-plausible_cc196972d29c2bf393ef85b23518c86a87708716         |   |
| patch1-Math-32-JGenProg2017_5b35da90da9501eda187fba8a5dac943c1a2c5f5           |   |
| patch1-Math-79-SimFix_5f57bcfae6e6bebc98de9d9b49d01e6240b63554                 |   |
| patch2-Math-41-Cardumen_820e0638360e991e139de9b1558aa293d099f8c7               |   |
| patch4-Math-70-Cardumen_6ae26132188f016d96b87875f419738e597c22ed               |   |
| patch4-Math-2-jMutRepair_bfdec12ac1a5fd449afcdc18f48dadd1d7b6a071              |   |
| patch3-Math-84-jMutRepair_577ca11d58b2fd1a1c7de2344d0f1bde748ea2fb             |   |
| patch5-Math-41-Cardumen_bce75e6cb469a02ec3b833c382547e5479b84431               |   |
| patch5-Math-50-JGenProg2017_9a2f466670f340a3703073addea1ba23be81d2c4           |   |
| patch3-Math-97-Cardumen_137fc2214bccd5e36c35db1a1262bd61100b867d               |   |
| patch2-Math-63-CapGen-plausible_8e0b207f9f8b916f673499376590543ddd3ac008       |   |
| patch1-Math-70-SketchFix_9c3a280921e577b0e232ac21c6bb08c9b4ab99e1              |   |
| patch4-Math-2-Arja-plausible_650f72623c02da89df5fa255cf2cb4d34a7de4f5          |   |
| patch3-Math-50-Jaid-plausible_b24919bc2e9247752bf80ce897fc691db0e77dcd         |   |
| patch1-Math-97-Cardumen_d3aa30af651cff0d89da4e2ce8e696453db94234               |   |
| patch4-Math-40-Cardumen_3e365d57d44d43fab05c8ed06657e3dd859e99b8               |   |
| patch1-Math-78-JGenProg2017_af25aacd0160dc271e611c8134ce3091650cfa7b           |   |
| patch3-Math-46-Cardumen_118d6472e4e88c6a6ad971b56ebc8393a5a7a34a               |   |
| patch6-Math-31-Arja-plausible_0077dfd7c6af8559c220c97ef5f073ed8b49af47         |   |
| patch2-Math-2-Cardumen_ec0623df748f0f7e78505c97283e88df4aea8e78                |   |
| patch2-Math-2-Arja-plausible_3afc4c1b3ac0b56c753112da0040cee7834833d6          |   |
| patch2-Time-17-Cardumen_6703ffcb27929203fe01bbac8b0c8193faa05184               |   |
| patch1-Math-2-Elixir-plausible_72a12117bb5b5f71f9b5a9a14cb0edc708b73d8a        |   |
| patch1-Math-50-Arja_18550a82d2268d7233b84c82622647333deba34d                   |   |
| patch3-Time-11-Arja-plausible_a8ad6b9853609e86cbf9534fff05377dfdf7790a         |   |
| patch4-Math-41-Cardumen_9f598c25fbf4391891141d21b3005635eb35857f               |   |
| patch5-Math-97-Cardumen_118303cc7ab959b864e6ad8704347ce7bcb4dd32               |   |
| patch1-Math-70-JGenProg2017_36ceb2ae32c903057a1f574e1e8963c3f26a5a2f           |   |
| patch4-Time-11-jKali_56dd0ed3cd92d9eca3ebff2780d0b0f4b70bc954                  |   |
| patch6-Math-2-Arja-plausible_e04f13412937616bb57e1b656fc01cc14ad92db6          |   |
| patch1-Math-31-Arja-plausible_5db7253811426931ad8b303027542106bbe756e5         |   |
| patch4-Math-84-jMutRepair_41c159bfacb60d21dd65adae0b84690afdce8337             |   |
| patch2-Math-73-Cardumen_f434c280ed4a5af886a69816008cd9c34317899a               |   |
| patch6-Math-63-CapGen-plausible_51bf90d32e1131415d4d3178088aba27e33d10ab       |   |
| patch3-Math-73-JGenProg2017_41d0a3f8612162a3575f8807ab851b805a2f9457           |   |
| patch3-Math-95-JGenProg2017_92430cd021f1afc55f40d99090893eddd620ed9b           |   |
| patch5-Math-78-JGenProg2017_9f0aa52e1e211974359e0b435f603e82f117b98a           |   |
| patch5-Math-95-JGenProg2017_30896a1448dbdb0482f4f469b943ffe6e162608b           |   |
| patch1-Math-49-JGenProg2015-plausible_afe6f5fb54375bb1f7bc025a0f1a6e3edda29341 |   |
| patch1-Math-58-SequenceR_18fd74c5c4eab6d002d968560f9cfe6bc63d1271              |   |
| patch3-Math-50-Cardumen_f1f3feb8c5e66c975113577de56c9fb44a891aa3               |   |
| patch1-Math-86-Arja_6bf70cd6214303b773d2091b60e93ed0a52caea1                   |   |
| patch1-Math-78-Cardumen_812e1a29f97386b9557de67127215ace3040f5f5               |   |
| patch2-Math-2-SOFix-plausible_77b1ff4ea9f57c49e7dae04c45a38e2c648d6777         |   |
| patch5-Time-17-Cardumen_4f2108495ab5f966ca753c39bed399ad4576cf79               |   |
| patch5-Math-78-Cardumen_8b033f93442e6730db11ae26fdde66ddc69012d1               |   |
| patch1-Math-70-Cardumen_ed4835446af20bee899cc647fa0b4ddb14c19970               |   |
| patch4-Math-30-Cardumen_64267fab82b0cf984698e07e86c5827d4ae093ec               |   |
| patch3-Math-2-jMutRepair_8cf6c2bf3361b35157416579533d8912f8ee1f5c              |   |
| patch1-Math-63-Cardumen_21dba3d0e53f1b2d8452aca3f760d418ff470cdc               |   |
| patch5-Math-70-Cardumen_af6373a9778a4579626928b9eee8a8719398a7e7               |   |
| patch4-Math-73-Cardumen_e8e34478c743ab5a538a78a08fa2d0c16abaf519               | #215   |
| patch1-Math-60-Arja-plausible_0d05fb6268f66414118ca29bbb794b6aaa1676ef         | #215  |
| patch1-Math-73-Elixir-plausible_aae15989859816d11726ca577e5d746b12911c92       |  #215  |
| patch5-Math-2-JGenProg2017_2cc0a7b1abd29d0148af87ae4e94175ad57309f8            | #218   |
| patch1-Math-70-SketchFix-plausible_39a8ce53a0ae54602713a6a679c4a0bed233ab12    | #217   |
| patch2-Math-46-Cardumen_049d3c1884211fb0d7a666773f1196ddad26a847               | Invisible Diff   |
| patch1-Time-19-SOFix_f2def9caae495f4193a57d42a6028c11163ddbfc                  | #216    |
| patch2-Math-78-JGenProg2017_622b83b6d32b541818ff2ce04c0584e9db976f3b           | #218   |
| patch1-Math-2-SOFix-plausible_1f4c3fea822e5d5d6c3b55af9b5f600f84e739d9         | #218    |
| patch1-Math-40-JGenProg2017_e624c905ff931b501d7c97fc03f37e66e09fb77d           | #215    |
| patch2-Time-4-Arja-plausible_edfc1fd057fdab1bfb06f04d51f57bb490630158          | #215 #219  |
| patch4-Math-50-Cardumen_3e4fa17a35acc1465b037e652bebd6697ec42a02               | #215   |
| patch8-Math-63-CapGen-plausible_3a36c5e8c60a87238df3450bd88c1177cfc6b76a       | #218    |
| patch3-Math-63-Cardumen_5b45b78464d5888ba60a3325f5ccc12876582cca               | #218   |
| patch1-Math-61-ACS_fbf87fd5c93e46b28c3987e16b05f1d1da0b021d                    |  #215  |
| patch2-Math-70-Cardumen_3177794e4cd6e9d09813d5e0c750c0cac5bc3da7               | #215   |
| patch15-Math-85-SequenceR_c54bd1816025f33a2ffaa092966e41f9b259f0dd             | https://github.com/ASSERT-KTH/collector-sahab/issues/221  |
| patch7-Math-2-Arja-plausible_b8b8b43c58e0804fb47a840cecd444e5be07622a          | #218    |
| patch1-Math-3-ACS_a7cc7ad18c379a784d5c97bbef7b073f82027825                     |  #215  |
| patch1-Math-2-jMutRepair_c869ef5c251cb9c94b14347cd918b9f2d921dc7e              | #218   |
| patch1-Math-50-jKali_d1dd4cc69d4b1f7dac12469ce653690f9bbe5c0d                  | #215   |
| patch1-Math-90-ACS_fdf73f8ceef21f7e3e244ece82811f1ab61040ae                    | Invisible Diff  |
| patch2-Math-78-Cardumen_43657f67421f79d1cc88cf1128c50ac654fe21ce               | #218   |
| patch5-Math-50-Cardumen_70d2fe684196ca1a29b3fa88ad58102effa02f20               | #215  |
| patch1-Math-99-ACS_fcadcfa2001ad4a86718e6a98856b332d37a4825                    | #217   |
| patch2-Math-40-Cardumen_9c32db9c49780f3ca394cf8437302d56af604cf3               | #215   |
| patch1-Math-2-Arja-plausible_9c2d18b38dd7df29612bb9888c59d8cf262a7977          | #218   |
| patch4-Math-73-JGenProg2017_544e05ee119f227a6aa00266a70353c5aaa89869           | #215    |
| patch5-Math-2-Arja-plausible_32aac6e3a55aaec51e83faaa0844142df8048579          |  #218  |
| patch5-Math-63-Cardumen_1827393f855be5fa4c173efcca1805712236770d               | #218   |
| patch4-Time-17-Cardumen_6fa845f549ce7445b490028128918b213124bdec               | #216   |

**Comment by algomaster99** [comment]
@khaes-kth I have outlined some reasons for the failures to render an HTML output. You may have a look at it.

**Comment by khesoem** [comment]
Thank you! I'll have look.

**Comment by algomaster99** [comment]
Debugged and narrowed down the failures further in https://github.com/ASSERT-KTH/collector-sahab/discussions/225.
